src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -89,25 +89,43 @@ class SaveInProgressIntro extends React.Component {
 
     if (login.currentlyLoggedIn) {
       if (savedForm) {
-        const lastUpdated =
-          savedForm.lastUpdated || _.get('metadata.lastUpdated', savedForm);
-        const savedAt = this.props.lastSavedDate
-          ? moment(this.props.lastSavedDate)
-          : moment.unix(lastUpdated);
-        const expiresAt = moment.unix(savedForm.metadata.expiresAt);
-        const expirationDate = expiresAt.format('MMMM D, YYYY');
-        const isExpired = expiresAt.isBefore();
+        /**
+         * lastSavedDate = JS time (ms) - always undefined?
+         * savedForms.lastUpdated = unix time (seconds)
+         * savedForms.metadata.expiresAt = unix time
+         * savedForms.metadata.lastUpdated = unix time
+         * savedForms.metadata.savedAt = JS time (ms)
+         */
+        const { metadata = {} } = savedForm;
+        const lastUpdated = savedForm.lastUpdated || metadata.lastUpdated;
+
+        let savedAt = '';
+        if (this.props.lastSavedDate) {
+          savedAt = new Date(this.props.lastSavedDate);
```

Comment:
Good catch - if `lastSavedDate` is never passed in as a prop maybe we can remove it from this forms component @micahchiang . This would be in another PR.

![image](https://user-images.githubusercontent.com/2008881/149577679-0fc101c7-84c6-41a4-86aa-5e9d5ddcc95f.png)

Reply:
I feel like it was originally meant to be used for unit tests, but wasn't.